### PR TITLE
Create SajjanYadav.mdx

### PR DESCRIPTION
**Pull request by SajjanYadav**

### CODE REVIEW

File: contribution/SajjanYadav.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<Tab title="JS"
+  <Skills title="javascript">
+    <Skill
+      name="javascript"
+      icon=
+      {
+        Javascript : <svg xmlns="http://www.w3.org/2000/svg" aria-label="JavaScript" viewBox="0 0 512 512" id="javascript"><rect width="512" height="512" fill="#f7df1e" rx="15%"></rect><path d="m324,370c10,17 24,29 47,29 20,0 33,-10 33,-24 0,-16 -13,-22 -35,-32l-12,-5c-35,-15 -58,-33 -58,-72 0,-36 27,-64 70,-64 31,0 53,11 68,39l-37,24c-8,-15 -17,-21 -31,-21 -14,0 -23,9 -23,21 0,14 9,20 30,29l12,5c41,18 64,35 64,76 0,43 -34,67 -80,67 -45,0 -74,-21 -88,-49zm-170,4c8,13 14,25 31,25 16,0 26,-6 26,-30V203h48v164c0,50 -29,72 -72,72 -39,0 -61,-20 -72,-44z"></path></svg>
+      }
+      level="Newbie" 
+    >
```
Comment (priyankarpal) [inline contribution/SajjanYadav.mdx:27]: Add this two lines after `>`

``` suggestion
    >
       </Skill>
  </Skills>
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

File: contribution/SajjanYadav.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<Tabs>
+
+<Tab title="JS"
+  <Skills title="javascript">
+    <Skill
+      name="javascript"
+      icon=
+      {
+        Javascript : <svg xmlns="http://www.w3.org/2000/svg" aria-label="JavaScript" viewBox="0 0 512 512" id="javascript"><rect width="512" height="512" fill="#f7df1e" rx="15%"></rect><path d="m324,370c10,17 24,29 47,29 20,0 33,-10 33,-24 0,-16 -13,-22 -35,-32l-12,-5c-35,-15 -58,-33 -58,-72 0,-36 27,-64 70,-64 31,0 53,11 68,39l-37,24c-8,-15 -17,-21 -31,-21 -14,0 -23,9 -23,21 0,14 9,20 30,29l12,5c41,18 64,35 64,76 0,43 -34,67 -80,67 -45,0 -74,-21 -88,-49zm-170,4c8,13 14,25 31,25 16,0 26,-6 26,-30V203h48v164c0,50 -29,72 -72,72 -39,0 -61,-20 -72,-44z"></path></svg>
```
Comment (sboy99) [inline contribution/SajjanYadav.mdx:24]: Pass only svg

Comment (sboy99) [reply contribution/SajjanYadav.mdx:24]: remove the Javascript: one

**Review by sboy99 — CHANGES_REQUESTED** [review]

**Review by Deva-1903 — APPROVED** [review]

**Review by sboy99 — APPROVED** [review]
LGTM 🐎

**Pull request merged**